web-client/src/presenter/state.js

```diff
@@ -330,6 +334,10 @@ export const baseState = {
     todaysOrdersSort: [],
   },
   showValidation: false,
+  tableSort: {
+    sortField: 'createdAt',
+    sortOrder: 'asc',
```

Comment:
feels nitpicky, but should this 'asc' be making use of  `web-client/src/presenter/presenterConstants.js`?

Reply:
You're correct. I'll address shortly

Reply:
... updating in future work.